server/core/peer.go

```diff
@@ -72,3 +72,75 @@ func CountInJointState(peers ...*metapb.Peer) int {
 	}
 	return count
 }
+
+// PeerInfo provides peer information
+type PeerInfo struct {
+	peerID       uint64
```

Comment:
Why not directly use metapb.Peer?

Reply:
We use `PeerInfo` to maintain the information like `readKeys` / `readBytes`.

Reply:
I mean
```
type PeerInfo struct {
	peer           *metapb.Peer
	readKeys       uint64
        ...
}
```

Reply:
That's ok to me.

Reply:
We don't need `FromMetaPeer` anymore